Frameworks/libmysqlclient/build-libmysqlclient.sh

```
@@ -44,7 +44,7 @@ fi
 
 export CC=/usr/bin/clang
 export CXX=/usr/bin/clang++
-cd $MYSQL_SRC
+cd "$MYSQL_SRC"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)